deploy/build/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM ubuntu:18.04
```

Comment:
This container takes quite a long time to build.
If we are simply using it to build our binaries, maybe we can push the image to some registry and just pull the image on demand.

We can then just keep the Dockerfile definition here as documentation, and make changes to it as necessary.

Reply:
yes, we can have a base image in public registry
The 1st error is caused by your user id not in docker group:
sudo usermod -aG docker $USER
newgrp docker